cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -105,13 +105,20 @@ bool MapBuilderBridge::HandleSubmapQuery(
   }
 
   response.submap_version = response_proto.submap_version();
-  response.cells.insert(response.cells.begin(), response_proto.cells().begin(),
-                        response_proto.cells().end());
-  response.width = response_proto.width();
-  response.height = response_proto.height();
-  response.resolution = response_proto.resolution();
+  if (response_proto.textures_size() == 0) {
```

Comment:
Sorry, did not realize this is answering a query, not receiving an empty answer:
```
CHECK(response_proto.textures_size() > 0) << ...
```
And include the ID in the error message to aid debugging should this crash?

Reply:
Done.